stompserver/stompserver.go

```diff
@@ -27,6 +27,10 @@ import (
 	lbstomp "github.com/vkuznet/lb-stomp"
 	// stomp library
 	"github.com/go-stomp/stomp"
+	// prometheus apis
+	"github.com/prometheus/client_golang/prometheus"
```

Comment:
Not knowing anything about go.... Why is this line necessary? I see the use of promauto and promhttp. I presume those things need this underlying library, but don't they bring it in themselves?

Reply:
Because the code you want to use comes from promtheus package itself, therefore you need to load the package (similar to import in python). I still think it is excessive since Prometheus by itself use standard http library and provide `promhttp.Handler` API which now is in a code.